### PR TITLE
Updates on Layer5 Website

### PR DESCRIPTION
**Pull request by hargunkaur286**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit f1a05075f58d49f0dc058741278e49647afe3d4c at: https://672cb2c411a6c85dc3d77276--layer5.netlify.app

File: src/sections/GCP-Diagram/diagram.js [hunk 1]
```diff
@@ -176,7 +176,7 @@ const Gcp = () => {
         </Row>
         <Row className="catalog">
           <Col md={4} className="diagram-detail">
-            <h2 className="heading">Extensive GCP Icon Library</h2>
+            <h2 className="heading">Extensive GCP icon library</h2>
```
Comment (vishalvivekm) [inline src/sections/GCP-Diagram/diagram.js:179]:
```suggestion
            <h2 className="heading">Extensive GCP component library</h2>
```

**Comment by l5io** [comment]
🚀 Preview for commit 162eb7ec0a64c8fb59a52df421c84e9a8986d834 at: https://672df4db3e32fe4b2c28a682--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 9b6dc0e15bd3b4c0c298feb84b887ee78de00f5b at: https://672e06cf338bf6660824d5f1--layer5.netlify.app

**Comment by vishalvivekm** [comment]
@hargunkaur286 can this branch be merged? or are you about to push changes?

**Comment by hargunkaur286** [comment]
> @hargunkaur286 can this branch be merged? or are you about to push changes?

yes, ready to push

**Comment by vishalvivekm** [comment]
> > @hargunkaur286 can this branch be merged? or are you about to push changes?
> 
> yes, ready to push

I see, you'll be submitting new PR for the remaining tasks ? Changing the count ?

**Review by vishalvivekm — APPROVED** [review]

**Pull request merged**